src/components/UI/atoms/Image/Image.spec.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { render, cleanup } from '<src>/helpers/testUtils';
+import Image from './Image';
+import 'jest-dom/extend-expect';
+
+afterEach(cleanup);
+
+const defaultProps = {
+  ImageUrl: 'https://assets-ouch.icons8.com/preview/501/57c82ec0-5e6d-423e-b594-5d93a5f5f44b.png',
+  width: '50%',
+  height: '50%',
+  altText: 'welcome',
+  boxShadow: 'image',
+  borderColor: 'default',
+  borderRadiusType: 'fullRadius',
+};
+
+
+describe('Image', () => {
+  it('should render the Image component with default prop', () => {
+    const { container } = render(<Image {...defaultProps}/>);
+    expect(container).toBeTruthy();
+  });
+
+  it('should render the Image component', () => {
+    const { container } = render(
+      <Image
+        ImageUrl='https://assets-ouch.icons8.com/preview/501/57c82ec0-5e6d-423e-b594-5d93a5f5f44b.png'
```

Comment:
Please pass the props in like we did in the backend by using a function. There is too much repetition here.
e.g.

```
cons setup = props = {
  const defaultProps = {
   `props you are passing in`
}

const utils = render(<Image {...defaultProps} {...props} />)

return {
 ...utils
}

}
```